docs/advanced.rst

```diff
@@ -210,6 +210,33 @@ each possible permutation of values. See
 `Test Permutations <tests/variables.html#permutations>`__
 for more info.
 
+Conditionals
+~~~~~~~~~~~~
+
+The ``only_if`` and ``not_if`` sections of the test config allow users
+to specify when a test should or should not run. The users would have one
+or both of the directives ``only_if`` and ``not_if`` followed by a dictionary.
+The dictionary key(s) would consist of variables that Pavilion could resolve
+and compare against the values following the key. An example could look like:
```

Comment:
The ``only_if`` and ``not_if`` sections of the test config allow users
to specify the conditions under which a test should run. Tests are
'SKIPPED' unless each of their `only_if` conditions (and none of their `not_if` conditions) match. 
The conditions are `key:value/s` pairs; the key is a Pavilion variable, and and the value/s are one 
or more items that the _resolved_ value of the Pavilion variable might match.